### DISCUSSION TITLE
October Update: node ace does not list all commands

### DISCUSSION THREAD

**Discussion by mastermunj** [adonisjs/core#1850] [Help] [answered]
After following the October Release guide to update the packages, I am facing peculiar problem with listing various commands.

When I run `node ace`, I do not see all available commands, rather see a truncated list with only default commands.

Apparently, when `ace-manifest.json` is copied on project root, proper command list is visible.

Please help debug and fix the issue.

**Comment by DraftProducts** [comment]
Hi @mastermunj, i had this problem too.
You have to execute your command from build folder: 
`node build/ace`

**Comment by LeCoupa** [comment]
Run `node ace generate:manifest` and all commands will appear

**Comment by mastermunj** [comment, marked as answer]
As @thetutlage says and I quote.

"What the recent updates (with in memory typescript compiler), there is no build folder during development. And hence the file has to be in the app root"

In nutshell, with build folder gone, `ace-manifest.json` is generated on root of the project. So, running following command does the job.

`node ace generate:manifest`
